src/network/netdev/bond.c

```diff
@@ -88,6 +88,12 @@ static int netdev_bond_fill_message_create(NetDev *netdev, Link *link, sd_netlin
                         return r;
         }
 
+        if (b->peer_notify_delay != 0) {
+                r = sd_netlink_message_append_u32(m, IFLA_BOND_PEER_NOTIF_DELAY, b->peer_notify_delay / USEC_PER_MSEC);
```

Comment:
should this use DIV_ROUND_UP()?

(just asking, i have no idea what would be better here: rounding down or rounding up)